src/privileged/FirefoxMonitor.jsm

```diff
@@ -136,8 +136,10 @@ this.FirefoxMonitor = {
       }
     }
 
-    XPCOMUtils.defineLazyPreferenceGetter(this, "breachListURL",
-      this.kBreachListURLPref, this.kDefaultBreachListURL);
```

Comment:
Let's go ahead and remove the mention of haveibeenpwned.com completely.

Reply:
done